health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -107,12 +107,18 @@ export const UICustomizations = {
           };
 
           const onActionSelect = (e) => {
-            if (e.name == "MP_ACTIONS_EDIT_SETUP") {
-              window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${1}&microplanId=${row.id}&campaignId=${
-                row.campaignDetails.id
-              }`;
-            }
-            if (e.name == "MP_ACTIONS_VIEW_SUMMARY") {
+            if (e.name === "MP_ACTIONS_EDIT_SETUP") { 
+              if(parseInt(row?.additionalDetails?.key)!==9){
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)